oie/ja-jp/Content/Topics/Browser-Plugin/plugin-installation.htm

```diff
@@ -222,251 +140,173 @@
                                     </div>
                                     <div class="replace_top_nav">
                                         <div class="navbar">
-                                            <div class="dropdown" data-mc-conditions="MultiProdPubish.DocMenu">
-                                                <button class="dropbtn">ドキュメント <i class="fa fa-caret-down"></i></button>
-                                                <div class="dropdown-content">
-                                                    <div><a href="https://help.okta.com/ja-jp/Content/Topics/ReleaseNotes/okta-relnotes.htm">リリース・ノート</a>
-                                                    </div>
-                                                    <div><a href="https://help.okta.com/ja-jp/Content/index.htm">製品ドキュメント</a>
-                                                    </div>
-                                                    <div><a href="https://developer.okta.com/documentation" target="_blank">開発者用ドキュメント<img src="../../Resources/Images/okta-assets/external-link-icon.svg" style="margin-left: 8px;" /></a>
-                                                    </div>
+                                            <div class="dropdown" data-mc-conditions="MultiProdPubish.DocMenu">                                                 <button class="dropbtn">ドキュメント <i class="fa fa-caret-down"></i></button>                                                 <div class="dropdown-content">
```

Comment:
Is it our post processing or did vendor return file with many new line symbols removed?